docs/sources/mimir/operators-guide/monitor-grafana-mimir/dashboards/ruler/index.md

```diff
@@ -17,3 +17,9 @@ The Ruler dashboard shows health and activity metrics for the ruler and object s
 The following example shows a Ruler dashboard from a demo cluster.
 
 ![Grafana Mimir ruler dashboard](mimir-ruler.png)
+
+Note: Even while operating in [Remote ruler mode]({{< relref "../../../architecture/components/ruler/#remote" >}}) you will still see values for the `Read from ingesters - QPS` and `Write to ingesters - QPS` panels.
+
+This is because the metrics are inclusive of intermediate services and are showing the values that ultimately reach the ingesters.
```

Comment:
```suggestion
This is because the metrics are inclusive of intermediate services and are showing the requests that ultimately reach the ingesters.
```

? maybe? up to you